tensorrt_llm/_torch/modules/fused_moe/moe_scheduler.py

```
@@ -0,0 +1,1104 @@
+    """Forward-execution strategy for ConfigurableMoE.
+
+    Stateless w.r.t. model configuration. Holds a back-reference to the
+    owning ``ConfigurableMoE`` and reads (but does not write) wrapper
+    state. See module docstring for the contract.
+    """
+
+    def __init__(self, moe: "ConfigurableMoE") -> None:
+        self.moe = moe
+
+    @abstractmethod
+    def forward(
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)